website/docs/dbt-cli/install/docker.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: "Docker"
+title: "Use Docker to install dbt"
```

Comment:
@jtcohen6 Most of these title changes are to help make our site a little more accessible to people with visual impairments. I haven't navigated it with Live Caption turned on (yet!), but I believe this will help.